src/content/structured/components/back-to-top/guidance.mdx

```diff
@@ -59,6 +70,17 @@ Consider using it on all apps to improve usability for everyone, including keybo
   caption="Display the back to top button when a page has been scrolled."
 />
 
+### Icon only
+
+The icon only back to top button should be used when a page is zoomed, to minimise overlapping content.
```

Comment:
```suggestion
The icon-only back to top button should be used when a page is zoomed, to minimise overlapping content.
```
When using "icon only" to describe the back to top button, [a hyphen should be used](https://www.grammarly.com/blog/hyphen/#1).

Reply:
Have now updated to include hyphen in all mentions of icon-only